src/main/java/net/dv8tion/jda/api/utils/MarkdownSanitizer.java

```diff
@@ -0,0 +1,431 @@
+/*
+ * Copyright 2015-2019 Austin Keener, Michael Ritter, Florian Spieß, and the JDA contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.dv8tion.jda.api.utils;
+
+import gnu.trove.map.TIntObjectMap;
+import gnu.trove.map.hash.TIntObjectHashMap;
+import net.dv8tion.jda.internal.utils.Checks;
+
+/**
+ * Implements and algorithm that can strip or replace markdown in any supplied string.
+ *
+ * @see #sanitize(String, net.dv8tion.jda.api.utils.MarkdownSanitizer.SanitizationStrategy)
+ */
+public class MarkdownSanitizer
+{
+    /** Normal characters that are not special for markdown, ignoring this has no effect */
+    public static final int NORMAL =     0;
+    /** Bold region such as "**Hello**" */
+    public static final int BOLD      = 1 << 0; // **x**
+    /** Italics region for underline such as "_Hello_" */
+    public static final int ITALICS_U = 1 << 1; // _x_
+    /** Italics region for asterisks such as "*Hello*" */
+    public static final int ITALICS_A = 1 << 2; // *x*
```

Comment:
Lets remove the trailing `// *x*` comment from these as the jdoc above them has the same example.